tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -4415,11 +4415,6 @@ Status ConvertFusedBatchNorm(OpConverterParams* params) {
   TF_RETURN_IF_ERROR(GetNodeAttr(attrs, "data_format", &data_format));
   TF_RETURN_IF_ERROR(GetNodeAttr(attrs, "is_training", &is_training));
 
-  if (data_format != "NCHW") {
-    return errors::Unimplemented(node_def.op(),
-                                 " only supports data_format=NCHW");
-  }
-
```

Comment:
This is a comment for line 4431 (posting here because I cannot add there). Please update the check of the channel dimension (tensor->getDimensions().d[1]), because the channel can be now at idx=3.

Reply:
Good catch, thanks. I updated the check and added an extra test case for that, as it wasn't being tested before on NHWC.